target/linux/ath79/dts/ar9344_mikrotik_routerboard-lhg-5nd.dts

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "ar9344_mikrotik_routerboard-16m-nor.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "mikrotik,routerboard-lhg-5nd", "qca,ar9344";
+	model = "MikroTik LHG 5nD platform";
```

Comment:
Since this has been reopened and needs to be updated anyway, I wonder why the model here is different from DEVICE_MODEL below?

Reply:
Mainly because I followed the pattern I saw wit hthe other mikrotik devices for both (to be specific: the `DEVICE_MODEL` string comes from the LHG2 and the `model` string is based on the string in `ar9344_mikrotik_routerboard-sxt-5n.dtsi` which the dts file for the LHG5 is mainly based on).
Do you have suggestion what string to use to unify the two?

Reply:
I'm sharing this here, just in case it can help in some way.

MikroTik often sells two different products with very similar hardware. Sometimes they add an extra Ethernet port, sometimes they only differ in the antenna gain, etc. I recently spotted this with the ipq40xx-based "SXTsq 5ac" and the "Disc Lite5 ac". In the case of this devices, under /sys/firmware/mikrotik/hard_config, you can see that the "board_identifier" is the same, whereas differences appear in "board_product_code" and "product_name", and sometimes in "hw_options".

My idea with the "platform" thing was to use this word only in the common .dtsi file shared by different devices with the same "board_identifier", for purely informative purpose inside the .dtsi file itself. Then, the actual .dts file for each specific device would override the model name.

I suggest to add support for the LHG 5 using the device name, on a single .dts file. Eventually, if a second device appears using the same "board_identifier", a common .dtsi can be created, and then use each .dts to just set each device-specific parameters.

Cheers!

Reply:
I set it now to be the same as DEVICE_MODEL.
There is a dtsi file where I tried to split out all the options that similar devices will probably share.